src/middlewares/localStorageMiddleware.jsx

```diff
@@ -1,5 +1,12 @@
+import { USER_SIGN_IN, USER_SIGN_UP } from "../ducks/userInfo/userInfoActions";
+
 export const localStorageMiddleware = (store) => (next) => (action) =>  {
+
     next(action);
+
+    if (action.type === USER_SIGN_IN || action.type === USER_SIGN_UP) {
+        localStorage.setItem("token", store.getState().userInfo.token);
+    }
```

Comment:
You are right we need to save token for F5 case, but why do you choose localStorage?